### PR TITLE
Add specific representation of a beach

### PR DESCRIPTION
**Pull request by jmcanterafonseca**

### CODE REVIEW

**Comment by jmcanterafonseca** [comment]
@gtorodelvalle r?

File: PointOfInterest/doc/introduction.md [hunk 1]
```diff
@@ -2,4 +2,17 @@
...
\ No newline at end of file
++ `PointOfInterest` : A harmonised geographic description of a point of interest. According to [Wikipedia](https://en.wikipedia.org/wiki/Point_of_interest)
+a point of interest, or POI, is a specific point location that someone may find useful or interesting. 
+
++ `Beach` : A harmonised description of a beach.
+[OpenStreetMap](http://wiki.openstreetmap.org/wiki/Tag:natural%3Dbeach) defines it as a
+loose geological landform along the coast or along another body of water consisting of sand, gravel,
+shingle, pebbles, cobblestones or sometimes shell fragments, etc.
+
++ `Museum` : A harmonised description of a museum.
```
Comment (gtorodelvalle) [inline PointOfInterest/doc/introduction.md]: Although not really related to this PR's content (I mean dealing about beaches), I guess it is OK this new `Museum` entry in the documentation, right, right? The question also applies to the `TouristInformationCenter` just below this one :) Thanks!

Comment (jmcanterafonseca) [inline PointOfInterest/doc/introduction.md]: yes, I have prepared this document with upcoming data models ;)

Comment (gtorodelvalle) [inline PointOfInterest/doc/introduction.md]: Thanks!!! Merging ;)

**Pull request merged**